opennms-webapp/src/main/webapp/errors/unknownexception.jsp

```diff
@@ -90,31 +92,11 @@ function toggleDiv(divName) {
 
 <%
 final StringBuilder stBuilder = new StringBuilder();
-
-if (exception != null) {
-  if (exception instanceof ServletException && ((ServletException)exception).getRootCause() != null) {
-    exception = ((ServletException) exception).getRootCause();
-  } else if (exception.getCause() != null) {
-    exception = exception.getCause();
-  }
-  stBuilder.append(exception.getClass().getName());
-  /*
-  NOTE: While this may look like a good idea, we want to omit the message
-  to avoid transmitting sensitive information like file paths and other internal
-  information on the error page. If the user needs more precise information,
-  they can look at the webapp logs for the complete stack trace.
-
-  String message = exception.getMessage();
-  if (message != null && message.length() > 0) {
-    stBuilder.append(": ").append(exception.getMessage());
-  }
-  */
-  stBuilder.append("\n");
-  for (StackTraceElement ste : exception.getStackTrace()) {
-    stBuilder.append("\tat ").append(ste.toString()).append("\n");
-  }
+boolean showStrackTrace = "true".equals(System.getProperty("org.opennms.ui.show_stacktrace"));
```

Comment:
I'd use `Boolean.getBoolean()` here instead of `System.getProperty()`

Reply:
done.